README.md

```diff
@@ -18,6 +18,11 @@ heroku config:add MAXMIND_DB_NAME=City
 heroku config:add MAXMIND_DB_NAME=Country
 ```
 
+Add your licence key, which is now a requirement for downloading even the free databases.
```

Comment:
This would deserve a link to https://www.maxmind.com/en/geolite2/signup and https://www.maxmind.com/en/accounts/current/license-key (or a link to the blog post explaining the change, https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-geolite2-databases/).